src/fixes.rs

```diff
@@ -52,8 +52,17 @@ pub fn run(warnings: &mut [Warning], lines: &mut Vec<LineEntry>) -> usize {
             .iter_mut()
             .filter(|w| w.check_name == fixer.name())
             .collect();
+
         if !fixer_warnings.is_empty() {
-            count += fixer.fix_warnings(fixer_warnings, lines).unwrap_or(0);
+            match fixer.fix_warnings(fixer_warnings, lines) {
+                Some(fixer_count) => count += fixer_count,
+                None => {
+                    for warning in warnings {
+                        warning.mark_as_unfixed();
+                    }
+                    return 0;
+                }
```

Comment:
```suggestion
                None => return count,
```
I think we should return the `count` variable here, because we already fixed previous lines and other warnings are unfixed by default.